spec/unit/ssl/validator_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'spec_helper'
 require 'puppet/ssl'
+require 'puppet_spec/ssl'
 
 describe Puppet::SSL::Validator::DefaultValidator do
+  include PuppetSpec::Files
   let(:ssl_context) do
     mock('OpenSSL::X509::StoreContext')
   end
 
+  before(:all) do
+    @pki = PuppetSpec::SSL.create_chained_pki
```

Comment:
Can this just be replaced by the `let(:pki)` below?

Reply:
I added this in because the `create_chained_pki` is expensive in terms of execution time; if it is run every test, this set of tests will take around 30 seconds instead of 2. The tests couldn't seem to reference pki without the lets, so I added that in. It might be better to change the tests to `@pki` and remove the lets to make it more simple and direct.